### DISCUSSION TITLE
Policy on blocking slow requests in Coraza WasmPlugin

### DISCUSSION THREAD

**Discussion by dspeg** [corazawaf/coraza#616] [Q&A]
What are the options regarding handling slow requests, for request header and body? Thanks.

**Comment by jcchavezs** [comment]
Could you elaborate a bit more on the use case? I don't think I get it.
What is a slow request? And what do we expect coraza to do about it?

**Comment by dspeg** [comment]
I'm talking about a DoS attack with lots of slow requests sent. I found some [article ](http://blog.nexusguard.com/preventing-slow-http-attacks#:~:text=Slow%20HTTP%20attacks%20are%20denial,the%20rest%20of%20the%20data.) on this.
